src/main.c

```diff
@@ -42,6 +43,7 @@
 #define SM_DBUS_INTERFACE "org.gnome.SessionManager"
 #define SM_CLIENT_DBUS_INTERFACE "org.gnome.SessionManager.ClientPrivate"
 
+
```

Comment:
Why adding here a second blank line?

Reply:
By accident, taking it out and will force push